src/Mod/Fem/femobjects/material_mechanicalnonlinear.py

```diff
@@ -81,7 +81,7 @@ def add_properties(self, obj):
             )
 
         if not hasattr(obj, "MaterialModelNonlinearity"):
-            choices_nonlinear_material_models = ["simple hardening"]
+            choices_nonlinear_material_models = ["isotropic hardening","kinematic hardening"]
```

Comment:
If you change the list of values in an `EnumerationProperty`, you must specify how previous projects should be restored.
In Python you can do this in the `onDocumentRestored` function, basically assigning the new list and selecting the default value.
This must also be taken into account for name and property type changes.

Usually enumeration values in Freecad start with a capital letter, but it is a matter of preference.

Reply:
All right, done.